.github/workflows/build-macOS11-GCS.yml

```diff
@@ -89,8 +89,49 @@ jobs:
           bootstrap_args="${bootstrap_args} --enable-release-symbols";
           source $GITHUB_WORKSPACE/scripts/ci/build_libtiledb.sh
 
-          # GCS unit tests are temporarily unsupported on CI for MacOS. Fake success with
-          # this echo.
+          BaseDir="$(pwd)"
```

Comment:
Can we move this section to a script?

Reply:
(and call it in all the yml files - code duplication getting a bit unwieldy in these files)